### PR TITLE
change session timeout to include checkboxes

### PR DESCRIPTION
**Pull request by velimd**
### Change description ###
changing the way session timeout is sending values due to incorrect data on checkboxes.

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/NFDIV-

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

**Review by linusnorton — APPROVED** [review]

**Comment by velimd** [comment]
closed due to 
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#browser_compatibility

File: src/main/assets/js/session-timeout.ts [hunk 1]
```diff
@@ -12,12 +12,19 @@ const saveBeforeSessionTimeout = async () => {
...
+      } else {
+        body[key] = value;
+      }
+    }
     await fetch(url, {
       method: 'POST',
       headers: {
         'Content-type': 'application/json',
-        'csrf-token': body._csrf as string,
+        'csrf-token': body['_csrf'] as string,
```
Comment (linusnorton) [inline src/main/assets/js/session-timeout.ts:27]: surely this is the same?

Comment (velimd) [reply src/main/assets/js/session-timeout.ts]: its not needed anymore, but ide was complaining it might not exist.

**Comment by velimd** [comment]
re opened because IE is dead and we dont need to support it

**Review by linusnorton — APPROVED** [review]

**Pull request merged**